manifests/j/Jellyfin/Server/10.9.0/Jellyfin.Server.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Jellyfin.Server
+PackageVersion: 10.9.0
+InstallerType: nullsoft
+ReleaseDate: 2024-05-12
+Installers:
+- Architecture: x64
+  InstallerUrl: https://repo.jellyfin.org/files/server/windows/latest-stable/amd64/jellyfin_10.9.0-1_windows-x64.exe

```

Comment:
Not Found (HTTP 404):
https://repo.jellyfin.org/files/server/windows/latest-stable/amd64/jellyfin_10.9.0-1_windows-x64.exe

I think the correct URL is:
https://repo.jellyfin.org/files/server/windows/latest-stable/amd64/jellyfin_10.9.1_windows-x64.exe

(let me know if this is not the correct place to post this)